onnx/test/test_backend_onnxruntime.py

```diff
@@ -248,8 +208,7 @@ def run_node(cls, node, inputs, device=None, outputs_info=None, **kwargs):
 )
 
 # The following tests are new with opset 19 and 20, or ai.onnx.ml 4
-if ort_version is not None and Version(ort_version) < Version("1.16"):
-    # version should be 1.15 but there is no development version number.
+if ort_version is not None and ort_version < Version("1.17"):
```

Comment:
Just my understanding: why is the number bumped from 1.16 to 1.17?

Reply:
I was using ort nightly (1.17) and it was still failing some tests in the original 1.17 section so I bumped all versions by 1. But I shouldn’t because these tests are expected to pass once 1.17 is released. I will revert the changes

Reply:
Done